root/edit/info.tt

```diff
@@ -1,7 +1,7 @@
 [% IF summary %]
     [% IF edit.status != 1 && edit.status != 2 %]
         <div class="edit-status">
-            [%- edit.status_name -%]
+            [%- l(edit.status_name) -%]
```

Comment:
Thanks, it is now done in https://github.com/metabrainz/musicbrainz-server/pull/387/commits/5e954125d9d77110b905bdeec7019e90d96c33c7 and https://github.com/metabrainz/musicbrainz-server/pull/387/commits/4c8dac89549dac469a202fa68f486aa1fcf679bc .